pod/perlref.pod

```diff
@@ -117,6 +117,16 @@ brackets:
         'Clyde' => 'Bonnie',
     };
 
+But be aware:
+
+    $folks{A} = {
+        'Adam'  => 'Eve',
+        'Clyde' => 'Bonnie',
+    };
+    $folks{B} = $folks{A};           # just pointers to the anonymous hash
+    %{ $folks{C} } = %{ $folks{A} }; # deep copy
```

Comment:
This is a shallow copy, not a deep copy.

Reply:
I think it's great you folks could put something better into that page than my version.

I even had my first accident today hitting the resolve button.